sbatches/farmshare/README.md

```diff
@@ -1,9 +1,11 @@
 # Farmshare
 
-1. Following the original tutorial - the container to use in Rice is located at /farmshare/home/classes/bioe/301p/ce/ces - maintained by Dr.Paul Nuyujukian of Department of Bioengineering.
-2. On Rice, type `model load singularity`, and then `singularity exec /farmshare/home/classes/bioe/301p/ce/ces jupyter notebook --generate-config`. 
-3. Choose a password and type and verify password, it will be stored and later used to access the notebooks. 
-4. Fill out the params.sh file like the original tutorial by running `bash setup.sh`.  Choose a port that is higher than 32768 for the tunnel to work. 
+1. Go to your home directory in rice. Type `module load singularity`.
+2. Type `singularity pull library://sohams/default/image:latest`. 
```

Comment:
Is this your image? Is there any reason 1. We can't be using a standard maintained jupyter image, and 2. That this URI can't just be specified in the recipe (it would pull once and then cache).

Reply:
? not quite following you here - according to the farmshare docs - the image must be copied over to the farmshare directories. Hence, why I had to use the singularity pull. 

So any image you have to pull into farmshare to begin with.

Reply:
Sorry my suggestion is to do a singularity pull of something like:

```
docker://jupyter/scipy-notebook
```
or whichever stack you think is best supporting the use case. If you just put this with the recipe command as the unique resource identifier Singularity should either pull the image or retrieve from the cache (try it).

Reply:
Ohh, got it!
That docker image does not work, but my singularity image works (after removing the image from my directory). 

I think knowing that my container works for this example is probably better than finding another container that might work or might not work - I actually maintain this container for my research so I am confident that this would be a good example. 
library://sohams/default/image:latest

Reply:
I just have been so far been removing and pulling the container every-time I have been updating it.

Reply:
Gotcha! I have one request - could you please give that image a meaningful name? default image latest is probably the more generic image I could imagine. The user should know generally what they are getting from the name (e.g., I'd except jupyter in there).

Reply:
Sure! Yes, I do agree that is very generic - changed it to farmsharejupyter.